rustls/src/verify.rs

```diff
@@ -559,10 +576,24 @@ impl AllowAnyAuthenticatedClient {
                 .iter()
                 .map(|r| r.subject().clone())
                 .collect(),
+            crls: Vec::new(),
             roots,
         }
     }
 
+    /// Update the verifier to validate client certificates against the provided DER format
+    /// unparsed certificate revocation lists (CRLs).
+    pub fn with_crls(
+        self,
+        crls: impl IntoIterator<Item = UnparsedCertRevocationList>,
+    ) -> Result<Self, CertRevocationListError> {
+        let crls = crls
```

Comment:
Style nit: not much point in binding this?

Reply:
Fixed. Thanks